west-european-politics.csl

```diff
@@ -0,0 +1,225 @@
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>West European Politics</title>
+    <id>http://www.zotero.org/styles/west-european-politics</id>
+    <link href="http://www.zotero.org/styles/west-european-politics" rel="self"/>
+    <link href="http://www.tandf.co.uk/journals/authors/style/reference/ref_fwep.pdf" rel="documentation"/>
+    <author>
+      <name>João Cancela</name>
+      <email>joaocancela@fcsh.unl.pt</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="political_science"/>
+    <issn>0140-2382</issn>
+    <eissn>1743-9655</eissn>
+    <summary>WEP</summary>
+    <updated>2015-12-30T10:11:08+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor">
+      <name delimiter="," and="text"/>
+      <label form="short" strip-periods="true" prefix=" (" suffix=".)"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="always" initialize="false" name-as-sort-order="first"/>
+      <label form="short" prefix=", "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text"/>
+      <et-al font-style="italic"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="URL">
+        <text variable="URL"/>
+        <group prefix=" (" suffix=")">
+          <text term="accessed" text-case="capitalize-first" suffix=" "/>
+          <date variable="accessed">
+            <date-part name="month" suffix=" "/>
+            <date-part name="day" suffix=", "/>
+            <date-part name="year"/>
+          </date>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="thesis">
+        <text variable="title"/>
+      </if>
+      <else-if type="bill book graphic legal_case legislation motion_picture song" match="any">
+        <text variable="title" font-style="italic"/>
+      </else-if>
+      <else-if type="report" match="any">
+        <group>
+          <text variable="title" prefix="‘" suffix="’, "/>
+        </group>
+        <group>
+          <text variable="collection-title"/>
+          <text variable="issue"/>
+          <text variable="abstract"/>
```

Comment:
"Abstract" was a mistake. The idea behind "issue" was citing the working paper issue,instead of only the title of the series (ie, Author X (2000). 'Working paper title', Name of the working paper series, Y, University Z. Maybe this was not the correct way to do it.